examples/applications/plot_model_complexity_influence.py

```diff
@@ -6,25 +6,42 @@
 Demonstrate how model complexity influences both prediction accuracy and
 computational performance.
 
-The dataset is the Boston Housing dataset (resp. 20 Newsgroups) for
-regression (resp. classification).
+We will be using two datasets:
+    - Diabetes dataset for regression.
+      This dataset consists of 10 measurements taken from diabetes patients.
+      The task is to predict the disease
+      progression
+    - 20 Newsgroups dataset for classification. This dataset consists of
+      newsgroup posts. The task is to predict on which topic (out of 20 topics)
+      the post is written on
+
+We will model the complexity influence on three different estimators:
+    - SGDClassifier (for classification data) which implements stochastic
+      gradient descent learning
+      :class:`sklearn.linear_model.SGDClassifier`
```

Comment:
```suggestion
    - :class:`~sklearn.linear_model.SGDClassifier` (for classification data) which implements stochastic
      gradient descent learning
      
```
Adding the `~` to the start means the rendered link text is just `SGDClassifier` and not the whole fun path